WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/compose/PrepublishingHomeSocialItem.kt

```diff
@@ -0,0 +1,127 @@
+package org.wordpress.android.ui.posts.prepublishing.home.compose
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+import org.wordpress.android.ui.compose.components.TrainOfIcons
+import org.wordpress.android.ui.compose.components.TrainOfIconsModel
+import org.wordpress.android.ui.compose.theme.AppTheme
+
+@Composable
+fun PrepublishingHomeSocialItem(
+    title: String,
+    description: String,
+    avatarModels: List<TrainOfIconsModel>,
+    modifier: Modifier = Modifier
+) {
+    SocialContainer(
+        avatarCount = avatarModels.size,
+        modifier = modifier
+            .fillMaxWidth()
+            .background(MaterialTheme.colors.surface)
+            .padding(16.dp),
```

Comment:
💡 Would it make sense to use `Margin.ExtraLarge` instead of `16.dp` directly?

Reply:
Good point! I also added an `ExtraSmall` value for `2dp`, matching what we have in our `dimens.xml`.